src/amo/components/AddonMoreInfo/index.js

```diff
@@ -148,15 +147,15 @@ export class AddonMoreInfoBase extends React.Component<InternalProps> {
       privacyPolicyLink: addon.has_privacy_policy ? (
         <Link
           className="AddonMoreInfo-privacy-policy-link"
-          href={`/addon/${addon.slug}/privacy/`}
+          to={`/addon/${addon.slug}/privacy/`}
```

Comment:
this means it goes through the router now, right?

Reply:
Yes